build/v1/types.go

```diff
@@ -1349,6 +1349,13 @@ type SecretSpec struct {
 	MountPath string `json:"mountPath" protobuf:"bytes,2,opt,name=mountPath"`
 }
 
+const (
+	// BuildVolumeMountPath is where user defined BuildVolumes get mounted
+	BuildVolumeMountPath = "/var/run/openshift.io/volumes"
```

Comment:
This appears to be a guarantee for builder pods that cannot change after we produce them because builder images are built to rely upon them.